### PR TITLE
Add e2e coverage analysis docs

### PR DESCRIPTION
**Pull request by Vacant2333**
**What type of PR is this?**
/kind documentation

**Which issue(s) this PR fixes**:
Part of #3685

### CODE REVIEW

File: test/e2e/coverage_docs/clusterpropagationpolicy_test.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+| Test the high explicit/low priority/implicit priority propagation for deployment | high explicit/low priority/implicit priority ClusterPropagationPolicy propagation testing | [Configure PropagationPolicy/ClusterPropagationPolicy priority](https://karmada.io/docs/next/userguide/scheduling/resource-propagating#configure-propagationpolicyclusterpropagationpolicy-priority) |
+| Test the same explicit priority propagation for deployment                       | same explicit priority ClusterPropagationPolicy propagation testing                       | [Choose from same-priority PropagationPolicies](https://karmada.io/docs/next/userguide/scheduling/resource-propagating#choose-from-same-priority-propagationpolicies)                                |
+
+#### Suggestions
+1. May need add the tase case when the [deployment updates](https://karmada.io/docs/next/userguide/scheduling/resource-propagating#update-deployment).
```
Comment (XiShanYongYe-Chang) [inline test/e2e/coverage_docs/clusterpropagationpolicy_test.md:28]: tase -> test

File: test/e2e/coverage_docs/clusterpropagationpolicy_test.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+|----------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Test the high explicit/low priority/implicit priority propagation for deployment | high explicit/low priority/implicit priority ClusterPropagationPolicy propagation testing | [Configure PropagationPolicy/ClusterPropagationPolicy priority](https://karmada.io/docs/next/userguide/scheduling/resource-propagating#configure-propagationpolicyclusterpropagationpolicy-priority) |
+| Test the same explicit priority propagation for deployment                       | same explicit priority ClusterPropagationPolicy propagation testing                       | [Choose from same-priority PropagationPolicies](https://karmada.io/docs/next/userguide/scheduling/resource-propagating#choose-from-same-priority-propagationpolicies)                                |
+
+#### Suggestions
```
Comment (XiShanYongYe-Chang) [inline test/e2e/coverage_docs/clusterpropagationpolicy_test.md:27]: How about naming this with TODO

File: test/e2e/coverage_docs/clusterpropagationpolicy_test.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+1. May need add the tase case when the [deployment updates](https://karmada.io/docs/next/userguide/scheduling/resource-propagating#update-deployment).
+2. May need add the test case for the **same implicit priority** propagation.
+3. May need add the test case for **delete** the clusterPropagationPolicy.
```
Comment (XiShanYongYe-Chang) [inline test/e2e/coverage_docs/clusterpropagationpolicy_test.md:30]: We can create an issue to track these tasks in the future.

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Good job! Thanks a lot~

File: test/e2e/coverage_docs/clusterpropagationpolicy_test.md (hunk 3 above)
Comment (Vacant2333) [reply test/e2e/coverage_docs/clusterpropagationpolicy_test.md:30]: yes~ sure, and its work in progressnow i will add more

**Comment by XiShanYongYe-Chang** [comment]
Thanks @Vacant2333 
The CI failed, can you help fix it?

/assign

**Comment by jwcesign** [comment]
Hi, @Vacant2333 ,
The ci is fixed, can you rebase the newest code?

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Thanks~
/lgtm
/approve

**Comment by XiShanYongYe-Chang** [comment]
> `cluster propagation policy` and `failover` e2e test coverage may need enhancement, can u help me confirm that and create the issue @XiShanYongYe-Chang , there are some TODO in that analysis files

Of course, how can I help you? Create an issue?

**Comment by Vacant2333** [comment]
> > `cluster propagation policy` and `failover` e2e test coverage may need enhancement, can u help me confirm that and create the issue @XiShanYongYe-Chang , there are some TODO in that analysis files
> 
> Of course, how can I help you? Create an issue?

yes, and those TODO may not correct, im cant confirm that , may u can confirm those

**Comment by XiShanYongYe-Chang** [comment]
> yes, and those TODO may not correct, im cant confirm that , may u can confirm those

Ok, I will do it later~

**Pull request merged**